va/http.go

```diff
@@ -591,16 +596,17 @@ func (va *ValidationAuthorityImpl) processHTTPValidation(
 	if err != nil && fallbackErr(err) {
 		// Try to advance to another IP. If there was an error advancing we don't
 		// have a fallback address to use and must return the original error.
-		if ipErr := target.nextIP(); ipErr != nil {
-			return nil, records, err
+		nextIPErr := target.nextIP()
```

Comment:
We now have a name conflict: for me this parses as "next ipError", not "nextIP error."

Let's call this `advanceIPError`.